landscape.yml

```diff
@@ -462,62 +449,63 @@ landscape:
             homepage_url: https://sysdig.com/use-cases/kubernetes-security
             logo: sysdig-secure.svg
             crunchbase: https://www.crunchbase.com/organization/sysdig
-      - subcategory:
-        name: Analytics & Monitoring
-        items:
-          - item:
-            name: Datadog
-            homepage_url: https://www.datadoghq.com/
-            logo: datadog.svg
-            crunchbase: https://www.crunchbase.com/organization/datadog
-          - item:
-            name: Dynatrace
-            homepage_url: https://www.dynatrace.com/
-            logo: dynatrace.svg
-            crunchbase: https://www.crunchbase.com/organization/dynatrace-software
-          - item:
-            name: New Relic
-            homepage_url: https://newrelic.com/
-            logo: newrelic.svg
-            crunchbase: https://www.crunchbase.com/organization/new-relic
-          - item:
-            name: Sumo Logic
-            homepage_url: https://www.sumologic.com/
-            logo: sumologic.svg
-            crunchbase: https://www.crunchbase.com/organization/sumo-logic
   - category:
-    name: Change Management
+    name: Observability and Analysis
     subcategories:
       - subcategory:
-        name: Version Control
+        name: DevOps Metrics
         items:
-          - item:
-            name: Git
-            homepage_url: https://git-scm.com/
-            repo_url: https://github.com/git/git
-            logo: git.svg
-            crunchbase: https://www.crunchbase.com/organization/git
-          - item:
-            name: Gitea
-            homepage_url: https://gitea.io/en-us
-            repo_url: https://github.com/go-gitea/gitea
-            logo: gitea.svg
-            twitter: https://twitter.com/giteaio
-            organization:
-              name: Gitea
-          - item:
-            name: GitHub
-            homepage_url: https://github.com/
-            logo: git-hub.svg
-            crunchbase: https://www.crunchbase.com/organization/github
+        - item:
+          name: Four Keys
+          homepage_url: https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling
```

Comment:
I would use one of the following: 

- https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance
- https://cloud.google.com/devops/state-of-devops
- https://cloud.google.com/blog/products/devops-sre/announcing-dora-2021-accelerate-state-of-devops-report